src/core/blob.cpp

```diff
@@ -23,6 +23,25 @@ blob::~blob()
     delete backEnd_;
 }
 
+#ifdef SOCI_HAVE_CXX11
+
+blob::blob(blob &&other)
+{
+    *this = std::move(other);
+}
+
+blob &blob::operator=(blob &&other)
+{
+    backEnd_ = std::move(other.backEnd_);
```

Comment:
`move()` seems useless on a pointer:
```suggestion
    backEnd_ = other.backEnd_;
```

Reply:
It is, but I added it for semantic reasons. Plus, should we ever switch this to a smart pointer, we don't have to adapt the code here.